From 3d05c8998f6aad642174cba90bc8426eb9bc8be7 Mon Sep 17 00:00:00 2001
From: "Wang, Fei" <fwang12@ebay.com>
Date: Mon, 10 Mar 2025 11:30:52 +0800
Subject: [PATCH] [CELEBORN-1895] Bump log4j2 version to 2.25.3

### What changes were proposed in this pull request?

Bump log4j2 version to 2.25.3
https://github.com/apache/logging-log4j2/releases/tag/rel%2F2.25.3

### Why are the changes needed?
Bump to latest log4j2 bug fix release.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
GA.

Closes #3134 from turboFei/log4j2.

Authored-by: Wang, Fei <fwang12@ebay.com>
Signed-off-by: SteNicholas <programgeek@163.com>
---
 dev/deps/dependencies-server | 8 ++++----
 pom.xml                      | 2 +-
 project/CelebornBuild.scala  | 2 +-
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/dev/deps/dependencies-server b/dev/deps/dependencies-server
index 00a93015795..8f24085e36e 100644
--- a/dev/deps/dependencies-server
+++ b/dev/deps/dependencies-server
@@ -73,10 +73,10 @@ jetty-util/9.4.56.v20240826//jetty-util-9.4.56.v20240826.jar
 jsr305/1.3.9//jsr305-1.3.9.jar
 jul-to-slf4j/1.7.36//jul-to-slf4j-1.7.36.jar
 leveldbjni-all/1.8//leveldbjni-all-1.8.jar
-log4j-1.2-api/2.17.2//log4j-1.2-api-2.17.2.jar
-log4j-api/2.17.2//log4j-api-2.17.2.jar
-log4j-core/2.17.2//log4j-core-2.17.2.jar
-log4j-slf4j-impl/2.17.2//log4j-slf4j-impl-2.17.2.jar
+log4j-1.2-api/2.25.3//log4j-1.2-api-2.25.3.jar
+log4j-api/2.25.3//log4j-api-2.25.3.jar
+log4j-core/2.25.3//log4j-core-2.25.3.jar
+log4j-slf4j-impl/2.25.3//log4j-slf4j-impl-2.25.3.jar
 lz4-java/1.8.0//lz4-java-1.8.0.jar
 maven-jdk-tools-wrapper/0.1//maven-jdk-tools-wrapper-0.1.jar
 metrics-core/4.2.25//metrics-core-4.2.25.jar
diff --git a/pom.xml b/pom.xml
index fda2da5952f..081033fbf9e 100644
--- a/pom.xml
+++ b/pom.xml
@@ -88,7 +88,7 @@
     <guava.version>33.1.0-jre</guava.version>
     <junit.version>4.13.2</junit.version>
     <leveldb.version>1.8</leveldb.version>
-    <log4j2.version>2.17.2</log4j2.version>
+    <log4j2.version>2.25.3</log4j2.version>
     <lz4-java.version>1.8.0</lz4-java.version>
     <mockito.version>4.11.0</mockito.version>
     <mockito-scalatest.version>1.17.14</mockito-scalatest.version>
diff --git a/project/CelebornBuild.scala b/project/CelebornBuild.scala
index 741ad67ca7b..d29e3ce27c6 100644
--- a/project/CelebornBuild.scala
+++ b/project/CelebornBuild.scala
@@ -55,7 +55,7 @@ object Dependencies {
   // don't forget update `junitInterfaceVersion` when we upgrade junit
   val junitVersion = "4.13.2"
   val leveldbJniVersion = "1.8"
-  val log4j2Version = "2.17.2"
+  val log4j2Version = "2.25.3"
   val jdkToolsVersion = "0.1"
   val metricsVersion = "4.2.25"
   val mockitoVersion = "4.11.0"
